.gitignore

```diff
@@ -14,3 +14,4 @@ mkmf.log
 Gemfile.lock
 /yardoc
 /.yardoc
+.idea
```

Comment:
Can we get rid of this? I'd prefer to keep the IDE related stuff out of here.

Reply:
Definitely! @thestelz note that you can have a global ignore file for your own configuration:

```bash
echo '.idea' > ~/.gitignore 
git config --global core.excludesFile ~/.gitignore
```

See http://git-scm.com/docs/gitignore#_description for more

Reply:
@BuonOmo sorry, we all use the same ide, so it is habit to add it before committing. We don't use the global due to new hires needed to know to do that, so one less thing to slip through the cracks. I will, however, add it to mine so this doesn't happen in the future. Thanks for the info!